### DISCUSSION TITLE
windows7系统还是7.8.2版本，请问有更新吗？

### DISCUSSION THREAD

**Discussion by ycjdixon** [getlantern/lantern-forum-cn#408] [General]
windows7系统7.8.2版本，蓝灯极其不稳定，需要不断的“断开”、“连接”，才能勉强用上。还经常连不上。

**Comment by MaxMKMax** [comment]
@ycjdixon 

各位好! 这两个链接都可以下载. 这个版本 7.8.4 应该修复了闪退和别的问题了. 如果还有什么问题, 请再来反馈.
谢谢你们耐心的等待!

https://github.com/getlantern/lantern/blob/devel/README.zh.md

https://github.com/getlantern/lantern

**Reply by ycjdixon** [reply]
WIN7系统用户已经被放弃了吗？故障频发那么长时间，有没有考虑补偿？

**Comment by ycjdixon** [comment]
WIN7系统 还是&nbsp;7.8.2版本  




蓝精灵
***@***.***







                       
原始邮件
                       
                     

发件人："Mike K"< ***@***.*** &gt;;

发件时间：2024/4/7 15:38

收件人："getlantern/lantern-forum-cn"< ***@***.*** &gt;;

抄送人："ycjdixon"< ***@***.*** &gt;;"Mention"< ***@***.*** &gt;;

主题：Re: [getlantern/lantern-forum-cn] windows7系统还是7.8.2版本，请问有更新吗？ (Discussion #408)




 
@ycjdixon
 
各位好! 这两个链接都可以下载. 这个版本 7.8.4 应该修复了闪退和别的问题了. 如果还有什么问题, 请再来反馈.
 谢谢你们耐心的等待!
 
https://github.com/getlantern/lantern/blob/devel/README.zh.md
 
https://github.com/getlantern/lantern

**Comment by ycjdixon** [comment]
我又重新下载安装了一遍WIN7版本的，还是7.8.2.一如既往的无法使用，连上几秒后就断了。




蓝精灵
***@***.***







                       
原始邮件
                       
                     

发件人："Mike K"< ***@***.*** &gt;;

发件时间：2024/4/7 15:38

收件人："getlantern/lantern-forum-cn"< ***@***.*** &gt;;

抄送人："ycjdixon"< ***@***.*** &gt;;"Mention"< ***@***.*** &gt;;

主题：Re: [getlantern/lantern-forum-cn] windows7系统还是7.8.2版本，请问有更新吗？ (Discussion #408)




 
@ycjdixon
 
各位好! 这两个链接都可以下载. 这个版本 7.8.4 应该修复了闪退和别的问题了. 如果还有什么问题, 请再来反馈.
 谢谢你们耐心的等待!
 
https://github.com/getlantern/lantern/blob/devel/README.zh.md
 
https://github.com/getlantern/lantern
